lib/clearance/session.rb

```diff
@@ -115,11 +115,28 @@ def cookie_value
         value: remember_token
       }
 
-      if Clearance.configuration.cookie_domain.present?
-        value[:domain] = Clearance.configuration.cookie_domain
+      allowed_domains = Array(Clearance.configuration.cookie_domain).dup
```

Comment:
What do you think about moving the array wrapping to the reader in `Configuration`?